specification/_global/bulk/BulkRequest.ts

```diff
@@ -36,7 +37,7 @@ import { SourceConfigParam } from '@global/search/_types/SourceFilter'
  * @doc_id docs-bulk
  *
  */
-export interface Request<TSource> extends RequestBase {
```

Comment:
I don't think we should remove the generic. It's true that bulk can send widely different documents, but it can also send the same "structure" over and over.
I would keep it and let language generators add a default value for the generic.

Reply:
Good point, that makes sense. I added a commit that restores them (and adds the `TPartialDocument` for the new `UpdateAction`)